R/utils.R

```diff
@@ -979,6 +979,11 @@ expose_stan_functions <- function(function_env, global = FALSE, verbose = FALSE)
     stop("Exporting standalone functions with external C++ is not available before CmdStan 2.32",
          call. = FALSE)
   }
+  if (!is.null(function_env$hpp_code) &&
+      !any(grepl("[[stan::function]]", function_env$hpp_code, fixed = TRUE))) {
```

Comment:
@andrjohns Is this an ok way to check whether there are any user defined functions?

Reply:
Yep, looks right to me